lib/src/component.dart

```diff
@@ -62,8 +62,10 @@ abstract class FluxComponent<ActionsT, StoresT> extends react.Component {
     Map<Store, Function> handlers = new Map.fromIterable(redrawOn(),
         value: (_) => (_) => redraw())..addAll(getStoreHandlers());
     handlers.forEach((store, handler) {
-      StreamSubscription subscription = store.listen(handler);
-      _subscriptions.add(subscription);
+      if (store != null) {
+        StreamSubscription subscription = store.listen(handler);
+        _subscriptions.add(subscription);
+      }
```

Comment:
It seems to me that if a store is null, it's more likely an error than an expected scenario where the store just hasn't been loaded yet.. should we print a warning to the console?